packages/atomic/cypress/fixtures/test-fixture.ts

```diff
@@ -169,6 +169,7 @@ export class TestFixture {
           {
             method: 'POST',
             url: '**/rest/search/v2?*',
+            times: 9999,
```

Comment:
I thought all search requests were being intercepted, but it turns out only the first one was. I added `times: 9999` to intercept all of them. I can make this optional if you prefer (or if this fails some tests).